test/widgets/board_test.dart

```diff
@@ -87,23 +87,38 @@ void main() {
     });
 
     testWidgets('play e2-e4 move by tap', (WidgetTester tester) async {
-      await tester.pumpWidget(
+        await tester.pumpWidget(
```

Comment:
This indentation change looks weird.

Reply:
huh, not sure what happened there, fixed

Reply:
Ah, there's not `dart format` check in `.github/workflows/test.yml`, we'll probably want to change that..?